taf/resources/pre-push

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Path to the TAF CLI executable
+TAF_CLI="taf"
+
+# Run the TAF validation command
+$TAF_CLI repo validate
```

Comment:
It can take a while to validate the repository starting from the beginning. The point here is to check if the new commits are valid before pushing. After each update, we store the last validated commit. You'll see that in the same directory as the authentication repository, there is a folder named after it, like `_law` where there is a `last_validated_commit` file which is updated every time the updater fetches new commits. It should be sufficient to validate from that commit. `validate` command has a `--from-commit` option. We could maybe add a flag, `--from-latest` and if it is provided when calling the command, call `validate_repository` with this commit. `AuthenticationRepository` class has a `last_validated_commit` that can be used instead of reading from this file directly. That would be a huge improvement of this hook from the usability perspective